docs/Documentation/Drivers/Classic/FrozenSoilOutputFile.md

```diff
@@ -2,7 +2,7 @@
 
 When the model is run with the frozen soil algorithms, a third output file is produced which contains soil thermal output parameters. Since the frozen soils algorithm only works in with the full energy balance, there is only one format for the output file.
```

Comment:
This comment is not directly related to this PR but just a thought - when the global file specifies `FROZEN_SOIL=TRUE` but `FULL_ENERGY=FALSE`, should it exit simulation? I think right now it still runs?